Readme.md

```diff
@@ -25,30 +26,17 @@ Whether you're a seasoned dancer or a self-proclaimed dance floor disaster, our
 
 Ready to bust a move? Let the awkwardness begin! 🎉
 
-### Table of Contents
-
-#### Explanation:
-Include a table of contents to help users navigate through your README easily. Each section in your README should have a corresponding link in the table of contents.
-
-```markdown
 ## Table of Contents
 
 - [Project Overview](#project-overview)
-  - [What is the Project Based On?](#what-is-the-project-based-on)
+    - [What is the Project Based On?](#what-is-the-project-based-on)
 - [Installation](#installation)
 - [Usage](#usage)
 - [Features](#features)
 - [Contributing](#contributing)
-```
 
 ### Installation
```

Comment:
Consider keeping the explanations for 'Installation' and 'Usage' sections. These sections are crucial for new users and providing detailed instructions can help them get started with the project more easily. [medium]